src/Ombi/ClientApp/src/app/my-nav/nav-search.component.html

```
@@ -1,5 +1,17 @@
-<form [formGroup]='searchForm'>
-    <mat-form-field floatLabel="never" style="width: 100%;">
-        <input id="nav-search"  autofocus="autofocus" matInput placeholder="{{'NavigationBar.Search' | translate}}"  formControlName='input'>
-    </mat-form-field>
+<form [formGroup]='searchForm' class="search-wrapper" [class.expanded]="isExpanded">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)